website/content/docs/upgrading/upgrade-to-1.14.x.mdx

```diff
@@ -14,7 +14,7 @@ Vault 1.13. **Please read carefully**.
 
 ## Feature deprecations and EOL
 
-### Duplicative Docker images
+### Duplicative docker images
```

Comment:
```suggestion
### Duplicative Docker images
```
Docker uses "Docker image" in their docs, so we should match their naming.

Reply:
Updated all references in https://github.com/hashicorp/vault/pull/21426/commits/52907a2176a569e3a6d2407f297892749aa5b67a